app/components/viral/sortable_list/list_component.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="<%= @system_arguments[:container_classes] %>">
+  <%= title %>
+  <ul
+    id="<%= id %>"
+    data-controller="sortable-list"
+    data-sortable-list-group-name-value="<%= group %>"
+    class="<%= @system_arguments[:list_classes] %>"
+    >
+    <% unless list_items.empty? %>
+      <% list_items.each do |list_item| %>
+        <li
+        id="<%= list_item %>"
+        class="
+        block
+        w-full
+        px-4
+        py-2
+        border-b
+        border-slate-200
+        cursor-move
+        hover:bg-slate-100
```

Comment:
Can you remove the hover: and dark:hover: classes as this is also causing some strange behaviour when dragging a list item from the available to selected columns. The next list item gets highlighted as you are dragging the previous item over

Reply:
Removed in eed8e25f89304aabec0116cd5d92acc7eab71a95